.travis.yml

```diff
@@ -15,7 +11,11 @@ matrix:
   # Disabled until gRPC supports PyPy.
   # - python: pypy
   #   env: TOXENV=pypy
-  - python: 3.5
+  - python: 3.6
```

Comment:
We ran into some trouble with pylint on 3.6, so we held this env back to 3.5. You might need to do the same.